README.md

```diff
@@ -314,12 +314,13 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 
 ## Industry :factory:
 
-| Company | Description | Locations |
-| :------ | :---------- | :-------- |
-| [Critical Manufacturing](https://www.criticalmanufacturing.com) [:rocket:](https://careers.criticalmanufacturing.com/) | Manufacturing business-critical applications. | `Maia` |
-| [FARO Technologies](https://www.faro.com/) [:rocket:](https://www.faro.com/about-faro/career/) | The world’s most trusted source for 3d measurement. | `Porto` |
-| [Malvern Panalytical](https://www.malvernpanalytical.com/) [:rocket:](https://spectris.wd3.myworkdayjobs.com/Malvern_Panalytical_Careers) | Laboratory analytical instruments design and manufacturing. | `Maia` |
-| [Relex Solutions](https://www.relexsolutions.com/) [:rocket:](https://www.relexsolutions.com/careers/) | The market-leading supply chain & retail planning platform. | `Lisboa` |
+| Company                                                                                                                                   | Description                                                 | Locations |
+| :------                                                                                                                                   | :----------                                                 | :-------- |
+| [Critical Manufacturing](https://www.criticalmanufacturing.com) [:rocket:](https://careers.criticalmanufacturing.com/)                    | Manufacturing business-critical applications.               | `Maia`    |
+| [FARO Technologies](https://www.faro.com/) [:rocket:](https://www.faro.com/about-faro/career/)                                            | The world’s most trusted source for 3d measurement.         | `Porto`   |
+| [Malvern Panalytical](https://www.malvernpanalytical.com/) [:rocket:](https://spectris.wd3.myworkdayjobs.com/Malvern_Panalytical_Careers) | Laboratory analytical instruments design and manufacturing. | `Maia`    |
+| [Relex Solutions](https://www.relexsolutions.com/) [:rocket:](https://www.relexsolutions.com/careers/)                                    | The market-leading supply chain & retail planning platform. | `Lisboa`  |
+| [Smartex](https://www.smartex.ai/) [:rocket:](https://smartexai.notion.site/Careers-202df8b49bc84e978bb93278e163d9ad)                     | AI for the Next Generation of Textiles.                     | `Porto`   |
```

Comment:
This is the only line you should have changed.
But as you can see in the diff, you are doing a lot of changes!

Can you please start again from the **latest version** of the project's master branch and add Smartex?